src/toMatchOneOf.test.ts

```diff
@@ -47,6 +47,10 @@ describe('toMatchOneOf', () => {
     expect(['abc', 'def']).toMatchOneOf(['ghi'])
   })
 
+  it('expect empty array is correct', () => {
+    expect([]).toMatchOneOf([])
```

Comment:
The test case you have added is slightly different from the one you reported
```suggestion
    expect([]).toMatchOneOf([[]])
```

Reply:
just force push again, the following is the correct test case
```
expect([]).toMatchOneOf([])
```